packages/perseus/src/components/__stories__/highlight-tooltip.stories.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import * as React from "react";
+import {action} from "@storybook/addon-actions";
+
+import HighlightTooltip from "../highlighting/ui/highlight-tooltip";
+
+export default {
+    title: "Perseus/Components/Highlight Toolip",
+};
+
+export const Main = (args: any): React.Node => (
```

Comment:
Suggestion: Would it be possible/valuable to add in a Story and StoryArgs type for better  flow and so we don't have to use "any" as a type here? It just pops out to me since I've been working on increasing flow coverage and this is usually a place where we would update it. (Totally no worries if this is not necessary; just working on sharing my thoughts on PRs more ^_^ )

```suggestion
type StoryArgs = {||};

type Story = {|
    title: string,
|};

export default {
    title: "Perseus/Components/Highlight Toolip",
}: Story;

export const Main = (args: StoryArgs): React.Node => (
```